crates/core/component/ibc/src/component/host_interface.rs

```diff
@@ -0,0 +1,10 @@
+use async_trait::async_trait;
+use cnidarium::StateRead;
+
+#[async_trait]
+pub trait HostInterface {
+    async fn get_chain_id<S: StateRead>(state: S) -> anyhow::Result<String>;
+    async fn get_revision_number<S: StateRead>(state: S) -> anyhow::Result<u64>;
+    async fn get_block_height<S: StateRead>(state: S) -> anyhow::Result<u64>;
+    async fn get_block_timestamp<S: StateRead>(state: S) -> anyhow::Result<tendermint::Time>;
```

Comment:
Do we need to add host defined client verification parameters like `max_time_per_block`?

Reply:
I was thinking of doing a second pass to extract all chain-specific calls into this trait, currently this is only extracting the penumbra-chain calls